app/src/main/java/com/guerinet/mymartlet/util/dbflow/DBUtils.java

```diff
@@ -49,7 +50,7 @@
      * @param <T>        Object type
      */
     public static <T extends BaseModel> void replaceDB(Context context, String dbName,
-            Class<T> type, List<T> newObjects, @Nullable Callback callback) {
+                                                       Class<T> type, List<T> newObjects, @Nullable Callback callback) {
```

Comment:
I would like to keep my formatting if possible. In this situation, I don't like aligning the start of the params because if the method declaration is too long you end up having very little room to declare your different params. 
I could add the formatting rules I use within AS if that would be helpful.